buildscripts/common.props

```diff
@@ -6,14 +6,17 @@
 		<RepositoryUrl>https://github.com/castleproject/Core</RepositoryUrl>
 		<BuildVersion>0.0.0</BuildVersion>
 		<BuildVersion Condition="'$(APPVEYOR_BUILD_VERSION)'!=''">$(APPVEYOR_BUILD_VERSION)</BuildVersion>
+		<BuildVersionMajor>$(BuildVersion.Split('.')[0])</BuildVersionMajor>
+		<BuildVersionMinor>$(BuildVersion.Split('.')[1])</BuildVersionMinor>
 		<AppendRuntimeIdentifierToOutputPath>false</AppendRuntimeIdentifierToOutputPath> 
 	</PropertyGroup>
 
 	<PropertyGroup>
 		<Product>Castle Core</Product>
-		<Version>$(BuildVersion)</Version>
-		<AssemblyVersion>$(BuildVersion)</AssemblyVersion>
 		<Authors>Castle Project Contributors</Authors>
+		<FileVersion>$(BuildVersion)</FileVersion>
+		<VersionPrefix>$(BuildVersion)</VersionPrefix>
+		<AssemblyVersion>$(BuildVersionMajor).$(BuildVersionMinor).0</AssemblyVersion>
```

Comment:
I thought we decided to do x.0.0.0? i.e. just major. If we do major.minor then we'll still get problems with very minor release which we've got right now.

> We used to do `x.x.0.0`, but `x.0.0.0` sounds even better. `AssemblyFileVersion` should still have the actual version. (https://github.com/castleproject/Core/issues/288#issuecomment-315387834)

Reply:
My bad, all fixed.